patches/minecraft/net/minecraft/world/level/block/PointedDripstoneBlock.java.patch

```diff
@@ -0,0 +1,54 @@
+--- a/net/minecraft/world/level/block/PointedDripstoneBlock.java
++++ b/net/minecraft/world/level/block/PointedDripstoneBlock.java
+@@ -162,23 +_,12 @@
+ 
+    @VisibleForTesting
+    public static void m_221859_(BlockState p_221860_, ServerLevel p_221861_, BlockPos p_221862_, float p_221863_) {
+-      if (!(p_221863_ > 0.17578125F) || !(p_221863_ > 0.05859375F)) {
+          if (m_154203_(p_221860_, p_221861_, p_221862_)) {
+             Optional<PointedDripstoneBlock.FluidInfo> optional = m_154181_(p_221861_, p_221862_, p_221860_);
+             if (!optional.isEmpty()) {
+                Fluid fluid = (optional.get()).f_221893_;
+-               float f;
+-               if (fluid == Fluids.f_76193_) {
+-                  f = 0.17578125F;
+-               } else {
+-                  if (fluid != Fluids.f_76195_) {
+-                     return;
+-                  }
+-
+-                  f = 0.05859375F;
+-               }
+-
+-               if (!(p_221863_ >= f)) {
++               net.minecraftforge.fluids.FluidType.DripstoneDripInfo dripInfo = fluid.getFluidType().getDripInfo();
++               if (dripInfo != null && !(p_221863_ >= dripInfo.chance())) {
```

Comment:
This chance condition is backwards if you're describing it as the chance the cauldron will be filled. That should be `<`.

Reply:
I just replaced the old chance float with the one from the dripstone info here. The code works the same way it should (in that it checks that the random float is less than the chance, it's just that the original code uses greater than and inverts it. I can change this if desired, just wanted to keep it consistent.

Reply:
That's fine, but the javadocs should be updated then to reflect that.